### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by levogirar**
- Added admin dashboard
- Minor CSS fixes

### CODE REVIEW

File: WebRoot/Templates/navbar admin.php [hunk 1]
```diff
@@ -72,6 +82,6 @@
 		</div>
 	</section>
 	<?php endif ?>
-	<div id="version">Wuwana version 2.2.1</div>
+	<!-- <div id="version">Wuwana version 2.2.1</div> -->
```
Comment (Nils85) [inline WebRoot/Templates/navbar admin.php:85]: You can remove this line completely now we have this info in the "Dashboard" page (after login) in `WebRoot/admin/view.php`

File: WebRoot/admin/companies/view.php [hunk 2]
```diff
@@ -14,10 +14,8 @@
 			<?php include 'Templates/navbar admin.php' ?>
 		</nav>
 		<main>
-			<div class="information-error-box" style="display:none" id="MsgBox">
-				<div class="information-error-vertical"></div>
-				<h2 id="MsgTxt"></h2>
-			</div>
```
Comment (Nils85) [inline WebRoot/admin/companies/view.php:20]: This is used by the JavaScript scraper to display error messages if something happen. It's why the style is initially set to "display:none"
See `WebRoot/static/dhtml/admin.js`

File: WebRoot/admin/view.php [hunk 3]
```diff
@@ -14,27 +14,35 @@
...
+				</section>
+				<section>
+					<h2>Companies</h2>
+					<div class="box box-text">
+						<a href=""><p id="MsgTxt"></p></a>
+					</div>
+					<!-- <div class="information-error-box" style="display:none" id="MsgBox">
+					<div class="information-error-vertical"></div>
+					<a href=""><h2 id="MsgTxt"></h2></a>
+					</div> -->
```
Comment (Nils85) [inline WebRoot/admin/view.php:43]: This is used by the JavaScript auto-scraper to display error messages if something happen. It's why the style is initially set to "display:none"
See `WebRoot/static/dhtml/admin.js`

**Review by Nils85 — CHANGES_REQUESTED** [review]

File: WebRoot/admin/companies/view.php (hunk 2 above)
Comment (levogirar) [reply WebRoot/admin/companies/view.php:20]: I think it like to show on `admin/view.php` and for it to be visible.

Comment (Nils85) [reply WebRoot/admin/companies/view.php:20]: Yes, it is exactly the same system but on this page it's to display an error message if something happens when you add a new company

File: WebRoot/admin/view.php (hunk 3 above)
Comment (Nils85) [reply WebRoot/admin/view.php:43]: The JavaScript auto-scraper use the element with `id="MsgBox"` (to change `display:none`)...
So I don't know if `<div class="box box-text">` is supposed to be always displayed or not?

**Review by Nils85 — CHANGES_REQUESTED** [review]

File: WebRoot/admin/view.php (hunk 3 above)
Comment (levogirar) [reply WebRoot/admin/view.php:43]: I saw that the message shows: "scraping company name." I like seeing that message, even if now it doesn't work... 

`box-text` should be always displayed.

File: WebRoot/admin/companies/view.php (hunk 2 above)
Comment (levogirar) [reply WebRoot/admin/companies/view.php:20]: I could re-add it to the `company view.php`

Comment (levogirar) [reply WebRoot/admin/companies/view.php:20]: I put a placeholder for now. I think i have a nice way to display it in the future. When we can add companies, i will make it better. I think it is good for now.

**Pull request merged**